app/scripts/lib/fxa-client.js

```diff
@@ -24,6 +24,30 @@ define(function (require, exports, module) {
     return $.trim(str);
   }
 
+  const CONTEXTS_REQUIRE_KEYS = [
```

Comment:
I have to admit that I'm not really happy with this solution. It may be better to remove the wantsKeys check from fxa-client.js and do this form within account.js. In general it seems like we shouldn't pass in a relier to the fxa-client, and fxa-client should be much smaller. That is a larger refactor that's out of scope for this PR.